lib/codegen/src/write.rs

```diff
@@ -141,7 +168,15 @@ pub fn write_ebb_header(
     writeln!(w, "):")
 }
 
-pub fn write_ebb(w: &mut Write, func: &Function, isa: Option<&TargetIsa>, ebb: Ebb) -> fmt::Result {
+pub fn write_ebb<
```

Comment:
For consistency with `decorate_function`, I think we should rename `write_ebb` to `decorate_ebb`.

(In the future we could add a `write_ebb` which would do what `write_function` does, passing in `write_function_plain`, if we were to need such a thing.)